### PR TITLE
rocksdb_lite: Fix

### PR DESCRIPTION
**Pull request by infinisil**
###### Motivation for this change
Didn't build before.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/libraries/rocksdb/default.nix [hunk 1]
```diff
@@ -68,7 +68,7 @@ stdenv.mkDerivation rec {
     echo "BUILD CONFIGURATION FOR SANITY CHECKING"
     cat make_config.mk
     mkdir -pv $static/lib/
-    mv -vi $out/lib/librocksdb.a $static/lib/
+    mv -vi $out/lib/${LIBNAME}.a $static/lib/
```
Comment (Infinisil) [inline pkgs/development/libraries/rocksdb/default.nix]: The problem was that with `enableLite`, the library was called `librocksdb_lite.a` and not `librocksdb.a`, so the `mv` failed.

**Comment by Infinisil** [comment]
This was introduced with https://github.com/NixOS/nixpkgs/pull/36809

**Comment by Mic92** [comment]
@GrahamcOfBorg build rocksdb

**Comment by Mic92** [comment]
@Infinisil can you disable aarch64 support for now?

**Comment by Infinisil** [comment]
@Mic92 Done

**Comment by Mic92** [comment]
Is a backport required here?

**Comment by Infinisil** [comment]
@Mic92 Nope, it's all green on 18.03 https://hydra.nixos.org/job/nixos/release-18.03/nixpkgs.rocksdb_lite.x86_64-linux

**Pull request merged**